### PR TITLE
bpo-44249 Update README.rst 

### PR DESCRIPTION
**Pull request by Ayushparikh-code**
https://bugs.python.org/issue44249

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept this contribution by verifying everyone involved has signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).

## Recognized GitHub username

We couldn't find a [bugs.python.org](https://bugs.python.org/) (b.p.o) account corresponding to the following GitHub usernames:

@Ayushparikh-code

This might be simply due to a missing "GitHub Name" entry in one's b.p.o account settings. This is necessary for legal reasons before we can look at this contribution. Please follow [the steps outlined in the CPython devguide](https://devguide.python.org/pullrequest/#licensing) to rectify this issue.




You can [check yourself](https://check-python-cla.herokuapp.com/) to see if the CLA has been received.

Thanks again for the contribution, we look forward to reviewing it!

**Review by nanjekyejoannah — COMMENTED** [review]
This is too trivial for a news entry IMHO.

I also, forget if we can really merge changes if authors have not signed the CLA.

**Comment by Ayushparikh-code** [comment]
> This is too trivial for a news entry IMHO.
> 
> I also, forget if we can really merge changes if authors have not signed the CLA.

I signed CLA , look over it again

File: README.rst [hunk 1]
```diff
@@ -110,17 +110,17 @@ The entire Python directory is cleaned of temporary files that may have
...
-binary resulting from this step is not good for real life workloads as it has
+flags for each flavor. Note that this is just an intermediary step.  The
+binary resulting from this step is not good for real-life workloads as it has
 profiling instructions embedded inside.
 
 After the instrumented interpreter is built, the Makefile will run a training
-workload.  This is necessary in order to profile the interpreter execution.
-Note also that any output, both stdout and stderr, that may appear at this step
+workload.  This is necessary to profile the interpreter's execution.
+Note also that any output, both stdout, and stderr, that may appear at this step
```
Comment (zware) [inline README.rst]: This should not change; it's not a list, it's just a sub-section of the sentence.  I'm not sure this note is true anymore, though.

File: README.rst [hunk 2]
```diff
@@ -110,17 +110,17 @@ The entire Python directory is cleaned of temporary files that may have
...
 is suppressed.
 
 The final step is to build the actual interpreter, using the information
-collected from the instrumented one.  The end result will be a Python binary
+collected from the instrumented one.  The result will be a Python binary
```
Comment (zware) [inline README.rst]: I don't think this needs to change.

File: README.rst [hunk 3]
```diff
@@ -110,17 +110,17 @@ The entire Python directory is cleaned of temporary files that may have
...
-flags for each flavour. Note that this is just an intermediary step.  The
-binary resulting from this step is not good for real life workloads as it has
+flags for each flavor. Note that this is just an intermediary step.  The
+binary resulting from this step is not good for real-life workloads as it has
 profiling instructions embedded inside.
 
 After the instrumented interpreter is built, the Makefile will run a training
-workload.  This is necessary in order to profile the interpreter execution.
-Note also that any output, both stdout and stderr, that may appear at this step
+workload.  This is necessary to profile the interpreter's execution.
```
Comment (zware) [inline README.rst]: I'm not convinced by this change either.  Maybe
```suggestion
workload which is necessary to profile the interpreter's execution.
```

But I'm not sure any change is really needed.

File: Misc/NEWS.d/next/Documentation/2021-05-26-18-29-48.bpo-39798.D4UZnR.rst [hunk 4]
```diff
@@ -0,0 +1 @@
+updated and corrected readme with some grammatical mistakes
```
Comment (zware) [inline Misc/NEWS.d/next/Documentation/2021-05-26-18-29-48.bpo-39798.D4UZnR.rst]: This entry is definitely not needed, but NEWS entries should also have proper grammar :)

**Review by zware — DISMISSED** [review]

**Review by nanjekyejoannah — COMMENTED** [review]
@zware, please do the honors of closing the PR unless @Ayushparikh-code has any rebuttals for the changes proposed.

Edit: I think I misread some of the reviews I guess to mean the change wasn't necessary, my bad.

**Review by Ayushparikh-code — COMMENTED** [review]
Done

**Comment by Ayushparikh-code** [comment]
I have made the requested changes; please review again

**Review by Ayushparikh-code — COMMENTED** [review]
(●'◡'●)

File: README.rst [hunk 5]
```diff
@@ -110,17 +110,17 @@ The entire Python directory is cleaned of temporary files that may have
 resulted from a previous compilation.
 
 An instrumented version of the interpreter is built, using suitable compiler
-flags for each flavour. Note that this is just an intermediary step.  The
-binary resulting from this step is not good for real life workloads as it has
+flags for each flavor. Note that this is just an intermediary step.  The
+binary resulting from this step is not good for real-life workloads as it has
```
Comment (terryjreedy) [inline README.rst:114]: These two changes and one other are correct.  Will merge when CI done.

File: README.rst (hunk 1 above)
Comment (terryjreedy) [reply README.rst]: I am reverting except for `'s`.

**Review by terryjreedy — APPROVED** [review]

File: README.rst (hunk 2 above)
Comment (terryjreedy) [reply README.rst]: I was 50/50, so reverted.

File: Misc/NEWS.d/next/Documentation/2021-05-26-18-29-48.bpo-39798.D4UZnR.rst (hunk 4 above)
Comment (terryjreedy) [reply Misc/NEWS.d/next/Documentation/2021-05-26-18-29-48.bpo-39798.D4UZnR.rst]: I removed it.

**Comment by terryjreedy** [comment]
@nanjekyejoannah  Trivial changes like this, with no creative content, do not need a CLA,  But I still think getting it is better.  If nothing else, so the matter is taken care of for next suggestion.

**Review by zware — APPROVED** [review]

**Pull request merged**